java/dagger/BUILD

```diff
@@ -45,3 +45,8 @@ javadoc_library(
     root_packages = ["dagger"],
     deps = ["//third_party:jsr330_inject"],
 )
+
+java_library(
+    name = "gwt",
+    resources = glob(["**/*.gwt.xml"]),
```

Comment:
Looks like these got lost?

Do the .gwt.xml files need to be in any particular file structure in the .jar? If not, then I think we should make this `glob(["gwt_resources/**/*.gwt.xml"]),` or something like that. Or, alternatively, `//java/dagger/gwt:gwt` with the resources in `java/dagger/gwt/resources/...`